app/_containers/Search/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import styles from "./index.module.scss";
+
+// Hi
+interface SearchBarProps {
```

Comment:
Prefer using `type` to `interface`.

https://www.totaltypescript.com/type-vs-interface-which-should-you-use

Reply:
And just `Props` is enough and let's use that for convention. We do not export it anywhere, so the name could be shorten.